Lib/test/test_pyexpat.py

```diff
@@ -465,7 +466,7 @@ def test_exception(self):
                                        "pyexpat.c", "StartElement")
             self.check_traceback_entry(entries[2],
                                        "test_pyexpat.py", "StartElementHandler")
-            if sysconfig.is_python_build():
+            if sysconfig.is_python_build() and not (sys.platform == 'win32' and platform.machine() == 'ARM'):
```

Comment:
I'd rather fix this than skip it, but I haven't figured out where to start looking yet.